Libraries/LibIPC/Message.cpp

```diff
@@ -59,4 +59,9 @@ ErrorOr<void> MessageBuffer::transfer_message(Transport& transport)
     return {};
 }
 
+/* static */
+RefPtr<MessageResolverState> MessageResolverState::create(u64 request_id)
+{
+    return RefPtr<MessageResolverState> { new MessageResolverState { request_id } };
```

Comment:
this leaks immediately. you have to `adopt_ref`. c.f. https://github.com/LadybirdBrowser/ladybird/blob/master/Documentation/SmartPointers.md#manual-construction-1